playbooks/robusta_playbooks/silence.py

```diff
@@ -69,6 +70,12 @@ class BaseSilenceParams(ActionParams):
     alertmanager_auth: Optional[SecretStr] = None
     grafana_api_key: str = None  # type: ignore
 
+    @validator("alertmanager_url", allow_reuse=True)
```

Comment:
I think `allow_reuse` is redundant (https://docs.pydantic.dev/usage/validators/#reuse-validators)

Reply:
I added it, because reloading playbooks runs the decorator multiple times